protocol/communities/manager.go

```diff
@@ -472,6 +476,61 @@ func (m *Manager) runENSVerificationLoop() {
 	}()
 }
 
+// This function is mostly a way to fix any community that is missing tokens in its description
+func (m *Manager) fillMissingCommunityTokens() error {
+	controlledCommunities, err := m.Controlled()
+	if err != nil {
+		m.logger.Error("failed to retrieve orgs", zap.Error(err))
+		return err
+	}
+
+	unlock := func() {
```

Comment:
Maybe lock/unlock in the loop?

Reply:
That was my first idea too, but I then realized that each community will take some time to run and since it's run in a separate thread, there is a (slim) chance that while one community is evaluated, the next one gets modified by another process and then we reach that next community in the loop and it's too late, it was modified.

Since I get all communities at the top, I feel it's safer to lock at the top too.